plugins/webgui/public/translate/en-US.js

```diff
@@ -44,18 +44,18 @@ module.exports = {
   '流量：': 'Data Usage:',
   '周期：': 'Duration:',
   '到期时间：': 'Due Date:',
-  '最近连接：': 'Recent Connection:',
+  '最近连接：': 'Last Connected:',
   '修改密码': 'Change Password',
-  '续费': 'Renew',
+  '续费': 'Recharge',
   '不限时': 'unlimited',
   '不限量': 'unlimited',
   '无': 'none',
   '或': 'or',
   '点击这里': 'Click Here',
-  '付款立即开通帐号': 'pay and assign an account now',
-  '点击二维码或者用移动设备扫描二维码可自动填充服务器信息': 'Click the QR code or scan it by smartphone to automate the retrieval of server information',
-  '目前该用户没有分配账号，请联系管理员处理': 'The current user has not been assigned an account, please contact the administrator',
-  '请选择续费周期：': 'Please choose the length of service:',
+  '付款立即开通帐号': 'pay and get an account now',
+  '点击二维码或者用移动设备扫描二维码可自动填充服务器信息': 'Click the QR code or scan it by smartphone to retrieve server information automatically',
+  '目前该用户没有分配账号，请联系管理员处理': 'No account available yet. Please contact the administrator.',
+  '请选择续费周期：': 'Service duration:',
```

Comment:
Yeah, I think this one is worth improving. But the original one doesn't seem to be nice either.